app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -95,10 +104,14 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope() {
 
     @Inject lateinit var vpnServiceStateStatsDao: VpnServiceStateStatsDao
 
+    @Inject lateinit var vpnDatabase: VpnDatabase
```

Comment:
I don't think we need to inject this since we are already injecting vpnServiceStateStatsDao above.

Reply:
goot point. Done